bears/installation_tool.py

```diff
@@ -0,0 +1,161 @@
+import argparse
+import importlib
+import re
+import subprocess
+import sys
+
+from coala_utils.string_processing.StringConverter import StringConverter
+from coala_utils.Question import ask_question
+from coalib.collecting.Collectors import get_all_bears_names
+from coalib.collecting.Importers import iimport_objects
+from coalib.misc.Shell import call_without_output
+from prompt_toolkit import prompt
+from prompt_toolkit.contrib.completers import WordCompleter
+from pygments.lexers import SqlLexer
+from pyprint.ConsolePrinter import ConsolePrinter
+
+
+def install_pip_package(package_name):
+    """
+    Uses call to install a PyPi package.
+
+    :param package_name: The package to be installed.
+    """
+    print('To install the dependencies correctly, we will need administrator '
+          'rights from you.')
+    subprocess.call(['sudo', sys.executable, '-m',
+                     'pip', 'install', package_name, '--upgrade'])
+
+
+def get_output(command):
+    """
+    Runs the command and decodes the output and returns it.
+
+    :param command: The command to be run.
+    :return:        The output of the command, decoded.
+    """
+    function = subprocess.Popen(command, stdout=subprocess.PIPE)
+    return function.communicate()[0].decode("utf-8")
+
+
+def check_failed_bears(bear_list):
+    """
+    Checks if any bear failed installation. Exits with 1 if so.
+
+    :param bear_list: List of the failed bears.
+    """
+    if bear_list:
+        print('Bears that failed installing:' + "\n".join(bear_list))
```

Comment:
I know I should have mentioned this before, but log this to `stderr` instead? Since it's actually an error. Just nitpicking from now on :P